dags/ods/post-event-report/dags/generate-post-event-report.py

```diff
@@ -0,0 +1,45 @@
+"""
+Airflow DAG to generate PyCon TW post event reports
+"""
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.bash_operator import BashOperator
+
+DEFAULT_ARGS = {
+    "owner": "tai271828",
+    "depends_on_past": False,
+    "start_date": datetime(2021, 1, 1),
+    "retries": 2,
+    "retry_delay": timedelta(minutes=5),
+    "on_failure_callback": lambda x: "Need to send notification to Discord",
```

Comment:
Will it be better if we add something like `#TODO` or open an issue for it?

Reply:
+1, added a sub-item in the corresponding issue https://github.com/pycontw/PyCon-ETL/issues/5

Reply:
As the issue has been created, I gonna resolve this one